src/Fixer/Operator/ConcatSpaceFixer.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\Operator;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\ConfigurationException\InvalidFixerConfigurationException;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ * @author SpacePossum
+ */
+final class ConcatSpaceFixer extends AbstractFixer
+{
+    private $fixCallback;
+
+    /**
+     * Configuration must have one element 'spacing' with value 'none' (default) or 'one'.
+     *
+     * @param array $configuration
+     */
+    public function configure(array $configuration = null)
+    {
+        if (null === $configuration) {
+            $this->fixCallback = 'fixConcatenationToNoSpace';
+
+            return;
+        }
+
+        if (!array_key_exists('spacing', $configuration)) {
+            throw new InvalidFixerConfigurationException($this->getName(), 'Missing "spacing" configuration.');
+        }
+
+        switch ($configuration['spacing']) {
+            case 'one':
+                $this->fixCallback = 'fixConcatenationToSingleSpace';
+
+                break;
+            case 'none':
+                $this->fixCallback = 'fixConcatenationToNoSpace';
```

Comment:
would be great to not call methods by string. really hard to perform static analysis then